src/mcp_server_obsidian_omnisearch/server.py

```
@@ -4,7 +4,7 @@
 import os
 
 
-def serve(obsidian_vault_path: str):
+def serve(host: str, port: str, base_path: str = None):
```

Ruff (0.14.8)

Unused function argument: `host`

(ARG001)